datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/dataplatforminstance/mappers/DataPlatformInstanceMapper.java

```diff
@@ -33,6 +47,39 @@ public DataPlatformInstance apply(@Nonnull final EntityResponse entityResponse)
     MappingHelper<DataPlatformInstance> mappingHelper = new MappingHelper<>(aspects, result);
     mappingHelper.mapToResult(DATA_PLATFORM_INSTANCE_KEY_ASPECT_NAME, this::mapDataPlatformInstanceKey);
 
+    final EnvelopedAspect envelopedDataPlatformInstanceProperties = aspects.get(Constants.DATA_PLATFORM_INSTANCE_PROPERTIES_ASPECT_NAME);
+    if (envelopedDataPlatformInstanceProperties != null) {
+      result.setProperties(mapDataPlatformInstanceProperties(
+              new DataPlatformInstanceProperties(envelopedDataPlatformInstanceProperties.getValue().data()), entityUrn)
+      );
+    }
+
+    final EnvelopedAspect envelopedOwnership = aspects.get(Constants.OWNERSHIP_ASPECT_NAME);
+    if (envelopedOwnership != null) {
+      result.setOwnership(OwnershipMapper.map(new Ownership(envelopedOwnership.getValue().data()), entityUrn));
+    }
+
+    final EnvelopedAspect envelopedTags = aspects.get(Constants.GLOBAL_TAGS_ASPECT_NAME);
+    if (envelopedTags != null) {
+      com.linkedin.datahub.graphql.generated.GlobalTags globalTags = GlobalTagsMapper.map(new GlobalTags(envelopedTags.getValue().data()), entityUrn);
+      result.setTags(globalTags);
+    }
+
+    final EnvelopedAspect envelopedInstitutionalMemory = aspects.get(Constants.INSTITUTIONAL_MEMORY_ASPECT_NAME);
+    if (envelopedInstitutionalMemory != null) {
+      result.setInstitutionalMemory(InstitutionalMemoryMapper.map(new InstitutionalMemory(envelopedInstitutionalMemory.getValue().data())));
+    }
+
+    final EnvelopedAspect statusAspect = aspects.get(Constants.STATUS_ASPECT_NAME);
+    if (statusAspect != null) {
+      result.setStatus(StatusMapper.map(new Status(statusAspect.getValue().data())));
+    }
+
+    final EnvelopedAspect envelopedDeprecation = aspects.get(Constants.DEPRECATION_ASPECT_NAME);
+    if (envelopedDeprecation != null) {
+      result.setDeprecation(DeprecationMapper.map(new Deprecation(envelopedDeprecation.getValue().data())));
+    }
+
```

Comment:
code style: would you mind updating these blocks to use the `mappingHelper` like we do for the key aspect above? a good example to follow would be `DatasetMapper.java` - using the `mappingHelper` to do this aspect check and then either setting the value or calling a mapping function like `mapDataPlatformInstanceKey` below

Reply:
I took `ContainerMapper.java` as example, as pointed out in the instructions.
I will have a look at `DatasetMapper.java` instead.

Reply:
addressed in 4730adbe7